### ISSUE TITLE
Default value for modbutton banning other than perm ban.

### ISSUE THREAD

**Issue by creesch** [toolbox-team/reddit-moderator-toolbox#109]
https://www.reddit.com/r/toolbox/comments/c0lgwf/importexport_mod_macros/erfdqli/

**Comment by SpyTec** [comment]
I wonder, should we have two settings? One user and one subreddit?

User one we default to lets say 7 days, subreddit one is unset. And subreddit takes precedent over user setting

That way we can have the 7 day default config value as this issue is intended but also set a default per subreddit

**Comment by creesch** [comment]
Yeah that sounds like an excellent idea. The subreddit one can be housed in the banmacros property obviously.
